.github/workflows/release-prepare.yml

```
@@ -214,7 +214,7 @@ jobs:
                 --force-tag-creation \
                 --force-branch-creation \
                 --additional-manifests="crates/test_utils/wasm/wasm_workspace/Cargo.toml" \
-                --allowed-semver-increment-modes="!pre_patch beta-dev" \
+                --allowed-semver-increment-modes="!pre_minor beta-dev" \
```

this change can be dropped

```suggestion
                --allowed-semver-increment-modes="!pre_patch beta-dev" \
```